### ISSUE TITLE
🧑‍💻: Adding the static data of Books

### ISSUE THREAD

**Issue by Sid-80** [Avdhesh-Varshney/chanakya-niti#181]
:red_circle: **Title** :  Static Data of Books
:red_circle: **Enhancement Aim** : Add static data
:red_circle: **Changes** : 

Adding the static data in the json format which will be received in future backend api

### Screenshots 📷
![image](https://github.com/Avdhesh-Varshney/chanakya-niti/assets/101277141/5643f667-fd15-4802-9c34-7444184598a2)

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name :  Siddharth
- What is your participant role? 
GSSOC

***********************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Sid-80** [comment]
@Avdhesh-Varshney  I am willing to work on this issue

**Comment by Avdhesh-Varshney** [comment]
@Sid-80 no, static addition of books pdfs or other things will increase the size of repo which is not required. So, wait for it, backend is under processing.

**Comment by Sid-80** [comment]
@Avdhesh-Varshney sure!!
